### ISSUE TITLE
Breakpoint exception setting ignores symbolOptions / justMyCode

### ISSUE THREAD

**Issue by vchirikov** [OmniSharp/omnisharp-vscode#4332]
## Environment data
`dotnet --info` output:  5.0.200-preview.20601.7
VS Code version: 1.52.1
C# Extension version:  ms-dotnettools.csharp-1.23.8
.omnisharp 1.37.6-beta.16

## Steps to reproduce
1. settings.json:

```js
{
    "csharp.unitTestDebuggingOptions": {
    "allowFastEvaluate": true,
    "enableStepFiltering": true,
    "logging": {
      "programOutput": true,
      "elapsedTiming": true,
      "engineLogging": false,
      "exceptions": true,
      "moduleLoad": false,
      "threadExit": false,
      "browserStdOut": false,
      "processExit": true
    },
    "requireExactSource": true,
    "suppressJITOptimizations": true,
    "justMyCode": true,
    "symbolOptions": {
      "searchPaths": [],
      "searchMicrosoftSymbolServer": false,
      "searchNuGetOrgSymbolServer": false,
      "moduleFilter": {
        "mode": "loadOnlyIncluded",
        "includeSymbolsNextToModules": false,
        "includedModules": [
          "MyUnitTests.dll",
        ]
      }
    }
  }
}
```
2. Enable `All exceptions`
![image](https://user-images.githubusercontent.com/40987940/103710445-19ce4700-4fc6-11eb-9780-5d61b3f2ee60.png)
2. Debug a test.
3. Debugger breaks on non-user code.
![image](https://user-images.githubusercontent.com/40987940/103710523-46825e80-4fc6-11eb-8a47-a6df4c457fd9.png)

## Expected  behavior

Breakpoint settings should use symbolOptions or justMyCode setting. Or maybe we can have a different checkboxes for this.

## Actual behavior

Symbols of `xunit.runner.visualstudio.dotnetcore.testadapter.dll` and `xunit.runner.utility.netcoreapp10.dll` are loaded and we see a breakpoint on exception inside the xunit code. I can't disable `All Exceptions` because if an exception is thrown in a test, we don't interrupt it with the `User-Unhandled Exceptions` setting.

**Comment by gregg-miskelly** [comment]
The fact that you are seeing source code means that the debugger did load symbols for that dll, and hence it is considered "your code" and the debugger will stop on exceptions in it. This may be because you set `"suppressJITOptimizations": true,`.

**Comment by vchirikov** [comment]
> The fact that you are seeing source code means that the debugger did load symbols for that dll, and hence it is considered "your code"

it's true, in fact I can't configure whether to load or not load these xunit dlls with `includedModules` and `excludedModules` settings

**Comment by gregg-miskelly** [comment]
Yes, I believe the XUnit dlls ship with embedded symbols, so the debugger will always use those, though maybe we shouldn't in your case.

Is there a reason you have `suppressJITOptimizations` on?

**Comment by vchirikov** [comment]
> Is there a reason you have suppressJITOptimizations on?
I don't have specific reason, other than to mimic to the defaults of Visual Studio

**Comment by gregg-miskelly** [comment]
That option defaults to off in both Visual Studio and VS Code.

**Comment by gregg-miskelly** [comment]
I should add: I generally recommend to NOT enable it. There are a few scenarios where it makes sense, but especially if you have JMC on, it can result in confusing behavior (and this bug is a perfect example of that).
